src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27935.cs

```diff
@@ -0,0 +1,31 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue27935 : _IssuesUITest
+	{
+		public Issue27935(TestDevice device) : base(device) { }
+
+		public override string Issue => "iOS: Rectangle that is invisible when page loads can never be made visible";
+
+		[Test, Order(1)]
+		[Category(UITestCategories.Shape)]
+		public void ShapesShouldAppearCorrectlyWhenIsVisibleChangesToTrue()
+		{
+			App.WaitForElement("Button");
+			App.Tap("Button");
+			VerifyScreenshot();
```

Comment:
Pending snapshots for macOS and Windows. Already available in the latest build:

For example:

![image](https://github.com/user-attachments/assets/17be049f-4429-4ad7-8168-8696bfbb1626)
![image](https://github.com/user-attachments/assets/24f95fe5-314e-486d-8678-5eee05fe6cb8)